instrumentation/instalogrus/instalogrus.go

```diff
@@ -0,0 +1,71 @@
+// (c) Copyright IBM Corp. 2021
+// (c) Copyright Instana Inc. 2021
+
+package instalogrus
+
+import (
+	instana "github.com/instana/go-sensor"
+	"github.com/opentracing/opentracing-go"
+	"github.com/sirupsen/logrus"
+)
+
+type hook struct {
+	sensor *instana.Sensor
+}
+
+// NewHook returns a new logrus.Hook to instrument logger with Instana
+func NewHook(sensor *instana.Sensor) *hook {
+	return &hook{
+		sensor: sensor,
+	}
+}
+
+// Levels returns the list of log levels to be sent to Instana
+func (h *hook) Levels() []logrus.Level {
+	return []logrus.Level{logrus.ErrorLevel, logrus.WarnLevel}
+}
+
+// Fire forwards the logrus.Entry to Instana
+func (h *hook) Fire(entry *logrus.Entry) error {
+	if entry.Context == nil {
+		h.sensor.Logger().Debug("ignoring logrus.Entry without context.Context")
+		return nil
+	}
+
+	parent, ok := instana.SpanFromContext(entry.Context)
+	if !ok {
+		return nil
+	}
+
+	msg, err := entry.Bytes()
```

Comment:
I think it can be simplified. What do you think? 
```suggestion
	msg, err := entry.String()
```